pkg/proc/job_lazy.go

```diff
@@ -52,87 +43,61 @@ func (job *Job) AssertStarted(ctx context.Context) error {
 	}
 }
 
-func (job *Job) start(ctx context.Context, process chan<- *os.Process) error {
-	l := log.WithField("job.name", job.Config.Name)
-
-	attempts := 0
-	maxAttempts := job.Config.MaxAttempts
-
-	if maxAttempts == 0 {
-		maxAttempts = 3
-	}
-
-	job.cmd = exec.Command(job.Config.Command, job.Config.Args...)
-	job.cmd.Stdout = os.Stdout
-	job.cmd.Stderr = os.Stderr
-	if job.Config.Env != nil {
-		job.cmd.Env = append(os.Environ(), job.Config.Env...)
-	}
-
-	for { // restart failed jobs as long mittnite is running
-		l.Info("starting job")
+func (job *LazyJob) Run(ctx context.Context, errors chan<- error) error {
+	listerWaitGroup := sync.WaitGroup{}
```

Comment:
did you mean `listenerWaitGroup`?

```suggestion
	listenerWaitGroup := sync.WaitGroup{}
```